proctor/src/main/java/TDS/Proctor/Sql/Repository/RemoteExamRepository.java

```
@@ -135,7 +135,7 @@ public void approveAccommodations(final UUID examId, final ApproveAccommodations
                 });
         } catch (HttpClientErrorException hce) {
             // No need to throw a ReturnStatusException if its a 4xx here - we'll leave it up to the service calling this method
-            if (isClientError(hce.getStatusCode())) {
+            if (hce.getStatusCode() == HttpStatus.UNPROCESSABLE_ENTITY) {
```

The status endpoint sends this specific error code when returning an body in the error.  Any other 400 would result in an Exception being thrown in the object mapper which is hiding the real error.